src/hooks/useBalanceChartData/useBalanceChartData.ts

```diff
@@ -46,7 +47,7 @@ export const priceAtBlockTime: PriceAtBlockTime = ({ time, assetPriceHistoryData
 }
 
 type CryptoBalance = {
-  [k: CAIP19]: number // map of asset to base units
+  [k: CAIP19]: BigNumber // map of asset to base units
```

Comment:
changing this to big number caught a bug with large erc20 sends and is more accurate